test/common/upstream/subset_lb_test.cc

```diff
@@ -373,6 +374,7 @@ class SubsetLoadBalancerTest : public testing::TestWithParam<UpdateOrder> {
   NiceMock<MockLoadBalancerSubsetInfo> subset_info_;
   std::shared_ptr<MockClusterInfo> info_{new NiceMock<MockClusterInfo>()};
   envoy::api::v2::Cluster::RingHashLbConfig ring_hash_lb_config_;
+  envoy::api::v2::Cluster::LeastRequestLbConfig least_request_lb_config_;
```

Comment:
The tests fail because this has default values (so 0 for choice_count), which would fail validation in a real config, but isn't checked in this test.